### PR TITLE
tt 1.0.0 (new formula)

### PR DESCRIPTION
**Pull request by better0fdead**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/tt.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  sha256 "6f42d30b9d9f9fbad1907a49cf3394f71c2c86e3faa1194fd9372e4d8877c792"
+  license "BSD-2-Clause"
+
+  depends_on "openssl@3" => :build
+  depends_on "mage" => :build
+  depends_on "go" => :build
+
+  def install
+    system "TT_CLI_BUILD_SSL=static mage build"
+    bin.install "tt"
```
Comment (SMillerDev) [inline Formula/tt.rb:14]: We don't want anything statically linked in Homebrew.

File: Formula/tt.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  end
+
+  test do
+    system "#{bin}/tt create cartridge --name cartridge_app -f --non-interactive -dst #{testpath}"
```
Comment (SMillerDev) [inline Formula/tt.rb:18]:
```suggestion
    system bin/"tt", "create", "cartridge", "--name", "cartridge_app", "-f", "--non-interactive", "-dst", testpath
```

File: Formula/tt.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+class Tt < Formula
+  desc "Command-line utility to manage Tarantool applications"
+  homepage "https://github.com/tarantool/tt"
+  url "https://github.com/tarantool/tt/releases/download/v1.0.0/tt-1.0.0-complete.tar.gz"
+  sha256 "6f42d30b9d9f9fbad1907a49cf3394f71c2c86e3faa1194fd9372e4d8877c792"
+  license "BSD-2-Clause"
+
+  depends_on "go" => :build
+  depends_on "mage" => :build
+  depends_on "openssl@3" => :build
```
Comment (SMillerDev) [inline Formula/tt.rb:10]:
```suggestion
  depends_on "openssl@3"
```

**Comment by better0fdead** [comment]
Sorry, i messed up. I close that PR. Will open another, so i won't annoy anyone.

**Comment by chenrui333** [comment]
#126546

**Pull request closed without merging**